Change default paths to Gentoo install locations.
Allow for compressed Info files.

--- ecb-2.32-orig/ecb-common-browser.el	2005-07-11 16:16:01.000000000 +0200
+++ ecb-2.32/ecb-common-browser.el	2009-03-03 09:42:51.000000000 +0100
@@ -261,10 +261,7 @@
 ;; additonal images are necessary - but currently i don't think we need
 ;; special images for this analyse-stuff.
 (defcustom ecb-tree-image-icons-directories
-  (let ((base (concat (if ecb-regular-xemacs-package-p
-                          (format "%s" (locate-data-directory "ecb"))
-                        ecb-ecb-dir)
-                      "ecb-images/")))
+  (let ((base "/usr/share/emacs/etc/ecb/ecb-images/"))
     (cons (concat base "default/height-17")
           (mapcar (function (lambda (i)
                               (cons (car i) (concat base (cdr i)))))
--- ecb-2.32-orig/ecb-help.el	2005-07-11 16:16:00.000000000 +0200
+++ ecb-2.32/ecb-help.el	2009-03-03 09:47:49.000000000 +0100
@@ -81,16 +81,8 @@
                  (const :tag "Html" :value html)))
 
 
-(defcustom ecb-help-info-path (concat
-                               (if ecb-running-xemacs
-                                   (if (file-exists-p
-                                        (concat ecb-ecb-dir
-                                                ecb-help-info-subdir
-                                                ecb-help-info-start-file))
-                                       ecb-help-info-subdir
-                                     "../../info/")
-                                 ecb-help-info-subdir)
-                               ecb-help-info-start-file)
+(defcustom ecb-help-info-path
+  (concat "/usr/share/info/" ecb-help-info-start-file)
   "*Path where the ECB online help in info format resides.
 This must be the location of the file \"ecb.info\" which comes with the ECB
 distribution. If is installed by unpacking the archive available on the ECB
@@ -107,24 +99,7 @@
   :type 'file)
 
 (defcustom ecb-help-html-path
-  (if (not ecb-running-xemacs)
-      (concat ecb-help-html-subdir ecb-help-html-start-file)
-    (cond ((file-exists-p
-            (concat ecb-ecb-dir
-                    ecb-help-html-subdir
-                    ecb-help-html-start-file))
-           (concat ecb-help-html-subdir ecb-help-html-start-file))
-          ((file-exists-p
-            (concat ecb-ecb-dir
-                    "../../html/"
-                    ecb-help-html-start-file))
-           (concat "../../html/" ecb-help-html-start-file))
-          ((file-exists-p
-            (concat ecb-ecb-dir
-                    "../../html/ecb/index.html"))
-           "../../html/ecb/index.html")
-          (t
-           (concat "../../etc/ecb/html/" ecb-help-html-start-file))))
+  (concat "/usr/share/doc/@PF@/html/" ecb-help-html-start-file)
   "*Path where the ECB online help in HTML format resides.
 This must be the location of the file \"index.html\" which comes with the ECB
 distribution. If is installed by unpacking the archive available on the ECB
@@ -145,7 +120,7 @@
   "Starts `info' with INFO-FILE. If INFO-FILE does not exists then nil is
 returned otherwise true. If NO-FILE-NOT-EXIST-ERR is not nil then just nil is
 returned if INFO-FILE does not exist otherwise an error is reported."
-  (if (file-exists-p info-file)
+  (if (file-expand-wildcards (concat info-file "*"))
       (prog1 t
         (info info-file))
     (unless no-file-not-exist-err
